saved-sites/saved-sites-impl/src/main/res/values/strings-saved-sites.xml

```diff
@@ -107,4 +107,11 @@
     <string name="deleteBookmarkFolderConfirmationDialogDescription" instruction="Placeholder is the name of the favorite site">This will delete your &lt;b>%1$s&lt;/b> bookmark folder.</string>
     <string name="deleteSavedSiteConfirmationDialogCancel">Cancel</string>
     <string name="deleteSavedSiteConfirmationDialogDelete">Delete</string>
+
+    <!--    Default Browser Bottom Sheet Dialog -->
```

Comment:
Did you mean adding the strings to the saves-sites module? I think they belong in the app module, like the rest of the code.

Also, as a good practice we want to add all the strings initially to a donottranslate.xml file. We do this until all the feature is completed, so we avoid generating translations. I’ll reach out to give you more context.

Reply:
No, thanks for pointing this out!